### PR TITLE
Run and Debug now uses dreamdaemon instead of dreamseeker

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Makes debug servers launch a dream daemon instead of a seeker.

## Why It's Good For The Game

There's some bad issues with seeker. It crashes randomly, and sometimes makes tgui run out of memory. Those problems aren't present when launching from daemon, so I think it's a good idea to switch over.

## Changelog

Not player facing... I really have no idea what tag to give this anyways.

### CODE REVIEW

**Comment by Kylerace** [comment]
dreamseeker is a piece of shit and i know of at least three people including me who put off work on a branch because ds kept crashing and they assumed it was their fault (which was fixed by switching to dreamdaemon).

File: .vscode/launch.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
       "request": "launch",
       "name": "Launch DreamSeeker",
       "preLaunchTask": "Build All",
-      "dmb": "${workspaceFolder}/${command:CurrentDMB}"
+      "dmb": "${workspaceFolder}/${command:CurrentDMB}",
+        "dreamDaemon": true
```
Comment (Kylerace) [inline .vscode/launch.json:10]: indenting

File: .vscode/launch.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
       "request": "launch",
       "name": "Launch DreamSeeker",
```
Comment (Kylerace) [inline .vscode/launch.json:7]:
```suggestion
      "name": "Launch DreamDaemon",
```

**Comment by tralezab** [comment]
Thank you Kyler

**Comment by Mothblocks** [comment]
ok this is actually rly annoying because it doesnt open anything and you have to manually connect. we should update the vsc extension to auto connect to the server

**Pull request merged**